test/Conversion/ExportVerilog/sv-dialect.mlir

```diff
@@ -1484,9 +1511,12 @@ hw.module @Verilator3405(
 
 // CHECK-LABEL: module CollectNamesOrder
 hw.module @CollectNamesOrder(%in: i1) -> (out: i1) {
-  // CHECK: wire _GEN;
-  // CHECK: wire [[EXTRA_GEN:.+]] = {{.+}};
-  // CHECK: assign {{.+}} = [[EXTRA_GEN]]
+  // OLD: wire _GEN;
+  // OLD: wire [[EXTRA_GEN:.+]] = {{.+}};
+  // OLD: assign {{.+}} = [[EXTRA_GEN]]
+  // FIXME: This is incorrect. `foo` should have the name "_GEN".
```

Comment:
Maybe link the issue you opened about this for now?

Reply:
Added, thanks!